shard-operator/pom.xml

```diff
@@ -146,6 +148,19 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>openshift</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <properties>
+        <namespace>default</namespace>
+        <quarkus.openshift.namespace>${namespace}</quarkus.openshift.namespace>
+        <quarkus.openshift.deploy>false</quarkus.openshift.deploy>
+        <quarkus.profile>openshift</quarkus.profile>
+        <event-bridge.k8s.platform>openshift</event-bridge.k8s.platform>
```

Comment:
Isn't this a duplication? As far as I can see we are declaring this property both via maven and via quarkus `application.properties` file
`%openshift.event-bridge.k8s.platform=openshift`

Reply:
It's a trick to set it in the maven.surfire.plugin . Otherwise running `mvn clean install -Popenshift` or mvn clean install -Popenshift` do not run the test suite with those values

Reply:
See a bit below in the file